eZ/Publish/SPI/Options/OptionsBag.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace eZ\Publish\SPI\Options;
+
+interface OptionsBag
+{
+    public function all(): array;
+
+    public function get(string $key, $default = null);
```

Comment:
Type hints for implicitly mixed param and return would be good here.

Reply:
Fixed :+1:. Obsolete comment.